examples/analysis/topoaa-caprieval_ensemble-test.cfg

```diff
@@ -0,0 +1,28 @@
+# ====================================================================
+# Capri analysis example
+
+# directory in which the scoring will be done
+run_dir = "run1-topoaa-caprieval_ensemble-test"
+
+# compute mode
+mode = "local"
+ncores = 10
+
+# ensemble to be analysed
+molecules = "data/ensemble_4G6M.pdb"
+
+# ====================================================================
+# Parameters for each stage are defined below, prefer full paths
+# ====================================================================
+[topoaa]
+
+[caprieval]
+# Comparison agains first reference
+reference_fname = "data/ensemble_4G6M.pdb"
```

Comment:
I find this example confusing for users: why using the same input ensemble as a reference in caprieval? the fact that we support this does not mean we should encourage this..this could become an integration test!

Reply:
Yes, indeed !
At least we see that we obtain 1 in dockq and 0 in rmsd.
I will find an other example !

Reply:
you can also remove the example and we only have this as integration test..depends on how much we want to advertise this feature

Reply:
OK